integrations/flink_connector/flink-connector-timestream/src/test/java/com/amazonaws/samples/connectors/timestream/SinkInitContext.java

```diff
@@ -60,6 +60,11 @@ public int getNumberOfParallelSubtasks() {
         return 0;
     }
 
+    @Override
+    public int getAttemptNumber() {
```

Comment:
Why is this change needed?

Reply:
Because we update the version of flink connector code to 1.7 and in that version org.apache.flink.api.connector.sink2.InitContext have new methods that should be implemented getAttemptNumber.